File: library/src/main/java/com/pengrad/telegrambot/model/request/Keyboard.java
package com.pengrad.telegrambot.model.request;

import com.google.gson.Gson;

import java.io.Serializable;

/**
 * stas
 * 8/11/15
 */
public abstract class Keyboard implements Serializable {
    private final static long serialVersionUID = 0L;

    // todo remove gson
    private static Gson gson = new Gson();

    @Override
    public final String toString() {
        return gson.toJson(this);
    }

}

File: core/src/main/kotlin/com/pengrad/telegrambot/model/request/Keyboard.kt
package com.pengrad.telegrambot.model.request

import com.google.gson.Gson

import java.io.Serializable

/**
 * stas
 * 8/11/15
 */
abstract class Keyboard : Serializable {

    override fun toString(): String {
        return gson.toJson(this)
    }

    companion object {
        private const val serialVersionUID = 0L

        // todo remove gson
        private val gson = Gson()
    }

}
